manual/src/tutorials/polymorphism.etex

```diff
@@ -26,9 +26,9 @@ let store = ref None ;;
 Since the type of "None" is "'a option" and the function "ref" has type
 "'b -> 'b ref", a natural deduction for the type of "store" would be
 "'a option ref". However, the inferred type, "'_weak1 option ref", is
-different. Type variables whose name starts with a "_weak" prefix like
-"'_weak1" are weakly polymorphic type variables, sometimes shortened as
-weak type variables.
+different. Type variables whose names start with a "_weak" prefix like
+"'_weak1" are weakly polymorphic type variables, sometimes shortened to
+"weak type variables".
```

Comment:
You probably meant ` ``weak type variables''` ( `"..."` is used for quoting code within the manual).

Reply:
Thanks. Fixed.